test/system/src/test/java/io/pravega/test/system/KeyValueStoreTest.java

```diff
@@ -0,0 +1,744 @@
+            assertNotNull("Get Entry List1 should not be empty", getEntryList1);
+            assertEquals("Get Entry List1 size should be same as get entry list1 size",
+                    getEntryList1.size(), entryMap1.size());
+            List<Integer> entryKeyList = new ArrayList<>();
+            for (Integer key: entryMap1.keySet()) {
+                entryKeyList.add(key);
+            }
+            log.info("entryKeyList: {}", entryKeyList);
+            for (int i = 0; i < keyArray.length; i++) {
+                assertEquals("Corresponding key for getEntryList should be as inserted",
+                        keyList.get(i), getEntryList1.get(i).getKey().getKey());
+                assertEquals("Corresponding key for entryList should be as inserted",
+                        entryKeyList.get(i), getEntryList1.get(i).getKey().getKey());
+            }
+            log.info("Corresponding keys for table entries are inserted");
+
+            /* Comparing each & every entry values of both entrylist and getEntryList */
+            for (int i = 0; i < keyArray.length; i++) {
+                assertTrue("All string values of both entryList1 and getEntryList1 should be equal",
+                        (entryMap1.get(entryKeyList.get(i))).equals(getEntryList1.get(i).getValue()));
+            }
+            log.info("All string values of both entryMap1 and getEntryList1 are equal");
+
+            boolean deleted = manager.deleteKeyValueTable(SCOPE_NAME, NEW_KVT_NAME);
+            assertTrue("The requested KVT must get deleted", deleted);
+            log.info("KVT '{}' got deleted", NEW_KVT_NAME);
+
+            /* Any operation performed over the above deleted Table must fail there upon */
+            List<TableEntry<Integer, String>> getEntryList2 = newKeyValueTable.getAll(keyFamily, keyList).join();
+            for (int i=0; i<getEntryList2.size(); i++) {
+                assertNull("All entries of getEntryList2 should be null", getEntryList2.get(i));
+            }
+            log.info("All entries of getEntryList2 are set to null");
+            log.info("Successfully completed test Delete KVT and check KVPs are not accessible");
+        }
+        catch (UnsupportedOperationException ex) {
+            throw new UnsupportedOperationException();
+        }
+        catch (Exception e) {
+            e.printStackTrace();
+        }
+    }
+
+    public Integer getKeyID() {
+        Integer keyId = ThreadLocalRandom.current().nextInt(1000,99999);
+        return keyId;
+    }
+
+    public String getValue() {
+        String value = "Hello World";
+        log.debug("Initial value length: {}", value.getBytes().length);
+        while (value.getBytes().length <= 10302) {
+           value+= ThreadLocalRandom.current().nextInt(1000, 9999);
+        }
+        log.debug("Final value length: {}", value.getBytes().length);
+        return value;
+    }
+
+    private static class IntegerSerializer implements Serializer<Integer> {
```

Comment:
This implementation is incorrect and is not needed. Use `io.pravega.test.integration.utils.IntegerSerializer`.

Reply:
okay, will be using this instead as suggested: 
```
private static class IntegerSerializer implements Serializer<Integer> {
        @Override
        public ByteBuffer serialize(Integer value) {
            ByteBuffer result = ByteBuffer.allocate(4).putInt(value);
            result.rewind();
            return result;
        }

        @Override
        public Integer deserialize(ByteBuffer serializedValue) {
            return serializedValue.getInt();
        }
    }
```
Is it correct ?